docusaurus/docusaurus.config.js

```diff
@@ -84,6 +79,12 @@ module.exports = {
   projectName: 'stream-chat',
   tagline: 'Stream Chat official component SDKs',
   themeConfig: {
+    algolia: {
+      apiKey: 'fd1b03de28081b5aa29dbccced4620b9',
+      indexName: 'DOCS',
+      appId: '7RY30ISS74',
+      // contextualSearch: true,
```

Comment:
This setup is needed in order to `show` the search component but it is not really used once we are using our own InstantSearch instead of DocSearch